internal/operators/cnv/cnv_operator.go

```diff
@@ -0,0 +1,135 @@
+package cnv
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/hardware"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	"github.com/openshift/assisted-service/internal/operators/lso"
+	models "github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	masterMemory int64 = 150
```

Comment:
Can you include the memory type ? Or else add a comment to state the type.
`masterMemory int64 = 150*units.MiB`
You'll also avoid having to do the conversion later on for each call to retrieve the master memory requirements.

Reply:
I added a comment with unit